src/modules/core-milestone/core-milestone.tsx

```diff
@@ -0,0 +1,289 @@
+/* eslint-disable no-confusing-arrow */
+
+import * as React from 'react';
+import {IProps} from './core-milestone.types';
+import Icon from '../icon/icon-external';
+import {minorScale} from '../scales';
+import Styled from 'styled-components/native';
+/* @ts-ignore */
+import ShapeHeart from '../../images/shape-heart.svg';
+/* @ts-ignore */
+import Baptism from '../../images/baptism.svg';
+/* @ts-ignore */
+import People from '../../images/people.svg';
+/* @ts-ignore */
+import ServingOpportunity from '../../images/serving-opportunity.svg';
+/* @ts-ignore */
+import ShapeExperiences from '../../images/shape-experiences.svg';
```

Comment:
I don't think we would need to ignore these? Also, can you please prepend these imports with the word "Icon" as in IconShapeHeart, IconBaptism.

Reply:
okay...i will update

Reply:
@ajaybushel build is fail if remove /* @ts-ignore */
can you pls help me

Reply:
@ajaybushel  can you please check above comments

Reply:
@hareshtech @yogeshpatel1404 its happening because TS is treating SVG files as modules. We had already added a declaration for SVG files in our `declarations.d.ts` file but it seems like each module should have its own declaration file. I have fixed the part and it should work now.